mssql_go18_test.go

```diff
@@ -0,0 +1,220 @@
+package odbc
```

Comment:
_go18 in a file name is pointless because it does not work - build fails. Even if you could make _go18 work, it is pointless, because I don't want to support go1.7 users. go1.7 is too old.

So, please, leave all tests in mssql_test.go.